ElementX/Sources/Screens/RoomScreen/View/TimelineView.swift

```diff
@@ -164,14 +164,14 @@ struct TimelineView: View {
 
 // MARK: - Previews
 
-struct TimelineTableView_Previews: PreviewProvider {
+struct UITimelineTableView_Previews: PreviewProvider {
```

Comment:
I think this should be?
```suggestion
struct TimelineView_Previews: PreviewProvider {
```